src/neuroconv/tools/nwb_helpers.py

```diff
@@ -184,3 +187,19 @@ def make_or_load_nwbfile(
 
                 if not success and not file_initially_exists:
                     nwbfile_path_in.unlink()
+
+
+def get_io_datasets(nwbfile):
+    for object_ in nwbfile.objects():
+        if isinstance(object_, TimeSeries) or isinstance(object_, VectorData):
+            for field in object_.__fields__:  # search for datasets
+                if field in ("data", "timestamps") and not isinstance(getattr(object_, field), DataIO):
```

Comment:
I was wondering about that, but can we for sure guarantee that the dataset of every child of `TimeSeries` will be under the `data` attribute?

Reply:
every descendent of TimeSeries should have a `data` field. That is a required argument. They may also have other datasets, and these other datasets would be missed. That could potentially be an issue if those other datasets are big, but usually they are just small metadata that you wouldn't want to compress anyway